flink-runtime/src/test/java/org/apache/flink/runtime/taskexecutor/TaskManagerRunnerTest.java

```diff
@@ -92,7 +92,7 @@ private static Configuration createConfiguration() {
 	}
 
 	private static TaskManagerRunner createTaskManagerRunner(final Configuration configuration) throws Exception {
-		TaskManagerRunner taskManagerRunner = new TaskManagerRunner(configuration, ResourceID.generate());
+		TaskManagerRunner taskManagerRunner = new TaskManagerRunner(configuration, ResourceID.generate(), null);
```

Comment:
why is this passing `null` and not following the same approach as `JobManagerHAProcessFailureRecoveryITCase`?

Reply:
adjusted